tasks/main.yml

```diff
@@ -27,6 +27,11 @@
   when: elasticsearch_file_name|get_major_version < 5
   failed_when: "'ERROR' in plugin_response.stdout and 'already exists' not in plugin_response.stdout"
 
+- name: Install mapper-size plugin on ES5
+  shell: /usr/share/elasticsearch/bin/elasticsearch-plugin install mapper-size
```

Comment:
Good point about second run. 

If we're allowing you to not install it, we should probably make a list of plugins to install that you can specify if you want. I'll do that!